packages/web-shared/components/AppHeader/AppHeader.js

```diff
@@ -0,0 +1,46 @@
+import Wordmark from '../Wordmark';
+import { useCurrentChurch } from '../../hooks';
+import { Box, Avatar } from '../../ui-kit';
+import { User } from '@phosphor-icons/react';
+
+import SearchStyles from '../Searchbar/Search.styles';
+
+import ProfileButton from '../Profile/ProfileButton';
+
+function ChurchLogo(props) {
+  const { currentChurch } = useCurrentChurch();
+
+  return <Wordmark source={currentChurch?.wordmarkLightUrl} padding={10} {...props} />;
+}
+
+const AppHeader = () => (
+  <>
+    <Box
+      py={8}
+      px={16}
+      height={58}
+      alignItems="center"
+      backgroundColor="fill.paper"
+      borderBottom="1px solid"
+      borderColor="text.quaternary"
+      position="fixed"
+      display="flex"
+      flexDirection="row"
+      justifyContent="space-between"
+      boxShadow="small"
+      top={0}
+      left={0}
+      right={0}
+      zIndex={999}
+    >
+      <Box>
+        <ChurchLogo size="30px" />
```

Comment:
There are some errors in the console about this prop needing to be a number on `Wordmark` if you want to look at that.

Reply:
Yeah, I saw that, but every other place we use `ChurchLogo` we pass a string like this. This code is getting trashed soon so I was going to keep it like we use it elsewhere and live with the warning